packages/braid-design-system/src/lib/components/Checkbox/Checkbox.tsx

```diff
@@ -17,17 +16,14 @@ export interface CheckboxProps
 }
 
 export const Checkbox = forwardRef<HTMLInputElement, CheckboxProps>(
-  ({ id, checked, tabIndex, ...restProps }, ref) => {
-    const resolvedId = useFallbackId(id);
-
+  ({ checked, tabIndex, ...restProps }, ref) => {
```

Comment:
If `InlineFIeld` no longer requires an `id` the props can probably be cleaned up too now.

```diff
export interface CheckboxProps
+  extends Omit<InlineFieldProps, 'checked'> {
-  extends Omit<InlineFieldProps, 'id' | 'checked'> {
-  id?: InlineFieldProps['id'];
  checked: CheckboxChecked | boolean[];
}
```